packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -316,6 +316,9 @@ export abstract class PersistentStream<
 
     this.cancelIdleCheck();
 
+    // Ensure we don't leave a pending backoff operation queued.
```

Comment:
This comment can move above the idle cancel and cover both:

```
// Ensure we don't leave pending timer-based operations queued.
```

Reply:
Hrm.  I had trouble creating a unified comment that I didn't feel was overly general or misleading, so instead I just commented both of them separately.

FWIW- the idle timer will only be active if the stream _is_ currently connected when close() is called, and the backoff timer will only be active if the stream _is not_ currently connected when close() is called.